docs/cli.md

```diff
@@ -591,3 +596,36 @@ gcli group grant --group groupA --role admin
 ```bash
 gcli group revoke  --group groupA --role admin
 ```
+
+### Topic commands
+
+#### Display a topic's details
+
+```bash
+gcli topic details --name kafka.default --topic topic3
```

Comment:
this should be in the same style as table's:
"gcli table details --name catalog_postgres.hr.departments"

Why not merge the "topic" name into "name" option?

Reply:
Yes, that is what I asked you about above.

Reply:
My comment is to keep consistent with table commands. As the entity name is different, users won't confuse.